### PR TITLE
[zeromq] Update to 2019-09-13

### PR DESCRIPTION
**Pull request by myd7349**

### CODE REVIEW

**Review by whidbey — APPROVED** [review]
I apply the patch and works ok.great man

**Comment by whidbey** [comment]
great ,works. thanks ~~

**Comment by vicroms** [comment]
@myd7349 looks like a PR to solve the file conflict issues has been merged, are you planning to update the PR to the latest commit on master?

File: ports/zeromq/portfile.cmake [hunk 1]
```diff
@@ -24,6 +24,7 @@ vcpkg_configure_cmake(
         -DBUILD_STATIC=${BUILD_STATIC}
         -DBUILD_SHARED=${BUILD_SHARED}
         -DWITH_PERF_TOOL=OFF
+        -DWITH_DOCS=OFF
```
Comment (JackBoosY) [inline ports/zeromq/portfile.cmake]: We should add docs as a feature.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/zeromq/portfile.cmake (hunk 1 above)
Comment (myd7349) [reply ports/zeromq/portfile.cmake]: Hi! @JackBoosY It seems that the docs are not generated correctly on Linux with asciidoc installed. In fact, I always got an empty `doc` dir. So, a `doc` feature will not work out.

**Comment by vicroms** [comment]
I prefer `websockets-sha1` and `websockets-nss` as the feature names.

File: ports/zeromq/CONTROL [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 Description: The ZeroMQ lightweight messaging kernel is a library which extends the standard socket interfaces with features traditionally provided by specialised messaging middleware products
 
 Feature: sodium
 Description: Using libsodium for CURVE security
 Build-Depends: libsodium
+
+Feature: websockets-sha1
+Description: Enable WebSocket transport through builtin sha1 (libzmq#3676)
+
+Feature: websockets-nss
```
Comment (vicroms) [inline ports/zeromq/CONTROL]: After some consideration, I think we shouldn't provide this feature until we can actually support it in vcpkg.

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/zeromq/CONTROL (hunk 2 above)
Comment (myd7349) [reply ports/zeromq/CONTROL]: @vicroms Thanks!

**Review by vicroms — APPROVED** [review]

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**